File: java-and-paradigms/code/parser/boolExpressions/java/pushingNegatives/token/Token.java
package pushingNegatives.token;

public class Token {
    private TokenType type;
    private char value;

    public Token(TokenType type, char value) {
        this.type = type;
        this.value = value;
    }

    public TokenType getType() {
        return type;
    }

    public char getValue() {
        return value;
    }
}

File: translation-methods/lab-2/src/main/kotlin/ru/itmo/m3334/skroba/tokenizer/Token.kt
package ru.itmo.m3334.skroba.tokenizer

import ru.itmo.m3334.skroba.tokenizer.TokenType.*

data class Token(val type: TokenType, val value: String = "#") {
    override fun toString(): String {
        return when (type) {
            NAME, TNAME -> value
            else -> type.toString()
        }
    }
}
